File: tmp-mei/fgenjs5030319945734616766/base/flows/_CheckFlowReuse_0.js
import * as rt from "../rt/main.js";
export class base$$flows$$_CheckFlowReuse_0 {
  static $self = new base$$flows$$_CheckFlowReuse_0();

  $hash$imm(isTail_m$) {
  return base$$flows$$_CheckFlowReuse_0.$hash$imm$fun(isTail_m$, this);
}

  static $hash$imm$fun(isTail_m$, $this) {
  return (function() {
  console.error("Program aborted at:\n" + new Error().stack);
  if (typeof process !== "undefined") process.exit(1);
  else throw new Error("Program aborted");
})()
;
}

}

File: tmp-mei/fgen4255027576711118260/base/flows/_CheckFlowReuse_0.java
package base.flows;
public interface _CheckFlowReuse_0{
_CheckFlowReuse_0 $self = new _CheckFlowReuse_0Impl();
base.Void_0 $hash$imm(base.Var_1 isTail_m$);
static base.Void_0 $hash$imm$fun(base.Var_1 isTail_m$, base.flows._CheckFlowReuse_0 $this) {
  return ((base.Void_0)(switch (1) { default -> {
  System.err.println("Program aborted at:\n"+java.util.Arrays.stream(Thread.currentThread().getStackTrace()).map(StackTraceElement::toString).collect(java.util.stream.Collectors.joining("\n")));
  System.exit(1);
  yield (base.Void_0) null;
}})
);
}
}